ADAL/src/telemetry/ADTelemetryDefaultEvent.m

```diff
@@ -132,13 +125,6 @@ - (NSString*)getStringFromDate:(NSDate*)date
     return [s_dateFormatter stringFromDate:date];
 }
 
-- (void)addAggregatedPropertiesToDictionary:(NSMutableDictionary*)eventToBeDispatched
-{
-    [eventToBeDispatched addEntriesFromDictionary:[self defaultParameters]];
-    SET_IF_NOT_NIL(eventToBeDispatched, AD_TELEMETRY_REQUEST_ID, [_propertyMap objectForKey:AD_TELEMETRY_REQUEST_ID]);
-    SET_IF_NOT_NIL(eventToBeDispatched, AD_TELEMETRY_CORRELATION_ID, [_propertyMap objectForKey:AD_TELEMETRY_CORRELATION_ID]);
-}
-
 - (NSDictionary*)defaultParameters
```

Comment:
This comment is on `-defaultParameters` which is an instance method. It should be a class method.